newfile.md

```diff
@@ -0,0 +1 @@
+# New file comment
```

Comment:
Nam adipiscing. Nam pretium turpis et arcu. Vestibulum volutpat pretium libero. Fusce risus nisl, viverra et, tempor et, pretium in, sapien.